### ISSUE TITLE
setup Fall 2020 session

### ISSUE THREAD

**Issue by shaolinhoward** [carrot-u/carrot-u-admin-app#86]
Well I lied, it's already there but we should revise the dummy lectures: https://carrot-u-admin.herokuapp.com/course_sessions/2

**Comment by shaolinhoward** [comment]
@shaolinhoward - look over student signup

**Comment by shaolinhoward** [comment]
work on description field

**Comment by shaolinhoward** [comment]
shoot for 10/5 announcement for next session @muffy @viktor-evdokimov

**Comment by shaolinhoward** [comment]
scope out Udemy for biz

https://github.com/carrot-u/carrot-u-docs/blob/master/2019-session/syllabus-2019.md

**Comment by shaolinhoward** [comment]
> @shaolinhoward - look over student signup

I totally forgot what this meant 😪 anybody remember? is this just to make sure student signup is working? emails sending out and so on?

**Comment by muffy** [comment]
Yes, we should make sure that student signup works from end to end (waiting list, session opened, manager and admin approval.

**Comment by shaolinhoward** [comment]
> scope out Udemy for biz
> 
> https://github.com/carrot-u/carrot-u-docs/blob/master/2019-session/syllabus-2019.md

started on this here: https://instacart.quip.com/45T3AWPCxLYi/Carrot-u-Syllabus-remapping
